Sources/GeneralIOs/Commands/Generate.swift

```diff
@@ -64,16 +69,26 @@ public final class Generate: ParsableCommand {
     }
 
     public func run() throws {
-        let renderer = Renderer(name: name,
-                                template: template,
-                                path: path,
-                                variables: [],
-                                output: output,
-                                dependencies: Services)
         if let xcodeSpec = generalSpec?.xcode {
-            guard let projectName = xcodeSpec.name ?? askProject() else {
+            guard let projectName = xcodeSpec.name ?? askProject(),
+                  let company = xcodeSpec.company ?? askCompany() else {
```

Comment:
is there a way get this value from template? it is seems method calls `askMissingVaariables`